transport/transporttest/reqres.go

```diff
@@ -95,7 +103,7 @@ func (m RequestMatcher) Matches(got interface{}) bool {
 		return false
 	}
 
-	if l.TTL != r.TTL {
+	if l.TTL-r.TTL > m.TTLDelta {
```

Comment:
nit: documentation makes it seem like this is an absolute delta, but the code shows that request TTL has to be <= the matcher's TTL within the delta.